drivers/hmis/app/models/hmis/hud/assessment.rb

```diff
@@ -66,6 +66,7 @@ def save_in_progress
     save!(validate: false)
     self.wip = Hmis::Wip.find_or_create_by(
       {
+        source: self,
```

Comment:
Fixes issue where we could only create 1 WIP assessment per enrollment per assessment_date